tests/integ/modin/index/test_datetime_index_methods.py

```diff
@@ -405,3 +405,97 @@ def test_floor_ceil_round_negative(func, freq, ambiguous, nonexistent):
         getattr(snow_index, func)(
             freq=freq, ambiguous=ambiguous, nonexistent=nonexistent
         )
+
+
+@pytest.mark.parametrize(
+    "native_index",
+    [
+        native_pd.date_range("2021-01-01", periods=5),
+        native_pd.date_range("2021-01-01", periods=5, freq="2D"),
+        pytest.param(
+            native_pd.DatetimeIndex(
+                [
+                    "2014-04-04 23:56:01.000000001",
+                    "2014-07-18 21:24:02.000000002",
+                    "2015-11-22 22:14:03.000000003",
+                    "2015-11-23 20:12:04.1234567890",
+                    pd.NaT,
+                ],
+                tz="US/Eastern",
+            ),
+            marks=pytest.mark.xfail(reason="Snowpark pandas does not support timezone"),
```

Comment:
make it more specific, to_datetime does not support tz.

Reply:
Can you create a todo ticket for me?

Reply:
https://snowflakecomputing.atlassian.net/browse/SNOW-1664175 here's the ticket!